### PR TITLE
feat: Bump to Beta 5

### PR DESCRIPTION
**Pull request by calldelegation**
Bumping up 
- [x] TS SDK v0.53.0
- [x] Rust SDK v0.69.1
- [x] Sway v0.48.1
- [ ] Redeploy new contracts on live beta-5 links

### CODE REVIEW

File: contract/Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ authors = ["sarahschwartz <58856580+sarahschwartz@users.noreply.github.com>"]
 license = "Apache-2.0"
 
 [dev-dependencies]
-fuels = { version = "0.47", features = ["fuel-core-lib"] }
+fuels = { version = "0.48.1", features = ["fuel-core-lib"] }
```
Comment (bitzoic) [inline contract/Cargo.toml:10]: This should be v0.53.0. The most recent version of the Rust SDK should resolve any issues you are running into using forc v0.48.1.

File: contract/fuel-toolchain.toml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "beta-4-rc.2"
\ No newline at end of file
+channel = "calldelegation_beta_5"
```
Comment (bitzoic) [inline contract/fuel-toolchain.toml:2]:
```suggestion
channel = "nightly-2023-12-06"
```
At least until beta-5 channel is released

File: contract/src/main.sw [hunk 3]
```diff
@@ -11,16 +11,15 @@ use {
             contract_id,
             msg_asset_id
         },
-        asset_id::{
-            construct_asset_id,
-        },
+        // contract_id,
```
Comment (bitzoic) [inline contract/src/main.sw:14]:
```suggestion
```

File: contract/src/main.sw [hunk 4]
```diff
@@ -85,7 +84,7 @@ impl GameContract for Contract {
 
     #[storage(read, write), payable]
     fn buy_seeds(food_type: FoodType, amount: u64) {
-        let asset_id_1 = construct_asset_id(contract_id(), ZERO_B256);
+        let asset_id_1 = AssetId::new(contract_id(), ZERO_B256);
```
Comment (bitzoic) [inline contract/src/main.sw:87]:
```suggestion
        let asset_id_1 = AssetId::default();
```
This newly introduced function makes reading this easier :)

**Comment by calldelegation** [comment]
Just need to redeploy contracts as soon as I have beta-5 Eth :)

**Review by sarahschwartz — APPROVED** [review]

**Pull request merged**